src/DataTable/CardView.jsx

```diff
@@ -37,21 +38,51 @@ function CardItem({
   return <CardComponent {...row} />;
 }
 
+function DefaultSkeletonCardComponent() {
+  return (
+    <Card isLoading>
+      <Card.ImageCap logoSkeleton />
+      <Card.Section className="pgn__data-table-card-view-default-skeleton-card-section" />
+      <Card.Footer />
+    </Card>
+  );
+}
+
 function CardView({
-  columnSizes, CardComponent, className, selectionPlacement,
+  columnSizes,
+  CardComponent,
+  className,
+  selectionPlacement,
+  SkeletonCardComponent = DefaultSkeletonCardComponent,
 }) {
   const {
     getTableProps, prepareRow, displayRows,
   } = useRows();
-  const { isSelectable, manualSelectColumn } = useContext(DataTableContext);
+  const {
+    isSelectable,
+    manualSelectColumn,
+    isLoading,
+  } = useContext(DataTableContext);
+
   // use the same component for card selection that is used for row selection
   // otherwise view switching might break if row selection uses component that supports backend filtering / sorting
-  const selectionComponent = manualSelectColumn?.Cell || selectColumn.Cell;
+  const SelectionComponent = manualSelectColumn?.Cell || selectColumn.Cell;
 
   if (!getTableProps) {
     return null;
   }
 
+  if (isLoading) {
+    return (
+      <CardGrid
+        className={classNames('pgn__data-table-card-view', className)}
+        columnSizes={columnSizes}
+      >
+        {[...new Array(8)].map(() => <SkeletonCardComponent key={nanoid()} />)}
```

Comment:
note: might consider allowing consumers to override the number of loading skeleton cards to display